### ISSUE TITLE
Composer network start/deploy appears to only honour the first -A definition

### ISSUE THREAD

**Issue by denisglotov** [hyperledger/composer#2688]
After moving to 0.15, what is the equivalent of `composer network start` where I could provide multiple admins (on 0.14):
```
# composer network start -a ../dist/my-network.bna -p mainorgtx \
#          -i MainPeerAdmin -s randomString \
#          -A admin-org1 -C ~/.identityCredentials/admin-org1-pub.pem \
#          -A admin-org2 -C ~/.identityCredentials/admin-org2-pub.pem

```
I tried (on 0.15.0)
```
composer network start -a ../dist/my-network.bna -c MainPeerAdminTx@my-network \
         -A admin-org1 -C ~/.identityCredentials/admin-org1-pub.pem \
         -A admin-org2 -C ~/.identityCredentials/admin-org2-pub.pem \
         -f /tmp/why_do_i_ever_need_this.card
```
but looks it does not work this way anymore 😞

## Context
To start the network I had to remove second '-A admin-org2 ...' line. Now I can start the rest server for org1, but for org2 I get this error:
```
$ composer-rest-server -c Org2CaAdmin@my-network -n never -w true -a true -p 3002
Discovering types from business network definition ...
Connection fails: Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity has not been registered:
admin-org2)
It will be retried for the next request.
Exception: Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity has not been registered: admin-org2)
Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity has not been registered: admin-org2)
    at _checkRuntimeVersions.then.catch (/home/denis/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:679:34)
```

I also tried to add the identity:
```
$ composer participant add -c MainCaAdmin@my-network -d '{"$class":"org.hyperledger.composer.system.NetworkAdmin", "participantId":"admin-org2"}'
Participant was added to participant registry.
                                 
Command succeeded                                                     
                                      
$ composer identity issue -c MainCaAdmin@my-network -u Admin2 -a org.hyperledger.composer.system.NetworkAdmin#admin-org2 -x true

Command succeeded

$ composer identity list -c MainCaAdmin@my-network | grep participant                                                                             ✔ List all identities in the business network
  participant: resource:org.hyperledger.composer.system.NetworkAdmin#admin-org2
  participant: resource:org.hyperledger.composer.system.NetworkAdmin#admin-org1
```
But the error with rest-server remained the same.

## Existing issues
This question on rocketchat: https://chat.hyperledger.org/channel/composer?msg=99kHarCWYkLwAeWPx and https://chat.hyperledger.org/channel/composer?msg=udLW9AqgbdKPYYNpA.

## Your Environment
```
$ composer-rest-server -v
v0.15.0
$ composer -v
v0.15.0
$ uname
Linux
```

**Comment by davidkel** [comment]
So this looks like only the first -A definition is now actioned. See #2665 as this is related

**Comment by mbwhite** [comment]
There is no problem with coding this; the only question is what does the -f option do?  It could specify multiple values - but how to align it to the -A.

One approach is, if there are multiple -A, then if the -f is a directory it writes out to that using the default name. If it's a file, then it uses that name, but adds a suffix with the value of -A  (before the extensions)

**Comment by mbwhite** [comment]
or pehaps more simply that the number of -A should match the number of -S/-C and -f.

**Comment by mbwhite** [comment]
resolved in #2614
